github-fetcher/json-orm_test.go

```diff
@@ -0,0 +1,292 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/google/go-github/github"
+)
+
+type issueTest struct {
+	gIssue        *github.Issue
+	expectedIssue *Issue
+}
+
+func makeIssue(number int,
```

Comment:
I completely agree, but I've done it for a good reason. I hate the idea of using a function to create the test that literally does the same as the tested function ...
But, because github.Issue takes pointer to everything, one can not do:
```
github.Issue{
    Number: 0 // It should take a pointer here, but &0 can't work for obvious reasons
}
```

So I would have to create a variable for each of these. To make it easier to get a variable, I used a function ...
Do you have a better way to do it? Otherwise I think I'll just remove the test ...